tuples.md

```diff
@@ -0,0 +1,67 @@
+# Writing Assignment - Tuples
+
+So far you have seen how to use arrays for holding a set of values and then iterate through them, for example:
+
+```swift
+var distances: [Int] = [10, 15, 20]
+
+for distance in distances {
+  print(distance)
+}
+```
```

Comment:
It's great that you establish what the students know by discussing arrays. But encasing this array within a concrete example helps ground this better for the student and create a narrative. For example, you can say. If I had a list of favorite movies, I can print them using the following array
(write code for that array here)